### PR TITLE
Removed sealed keywords from ColumnIndex and ColunmName attributes

### PR DESCRIPTION
**Pull request by yoldascevik**
It was created based on the topic #75.

### CODE REVIEW

**Comment by hughbe** [comment]
Are you able to add some tests, e.g. to `ExcelColumnIndexAttributeTests.cs`

```
[Theory]
[InlineData(0)]
[InlineData(1)]
public void Index_Set_GetReturnsExpectedInt(int index)
{
    var attribute = new ExcelColumnIndexAttribute(10)
    {
        Index = index
    };
    Assert.Equal(index, attribute.Index);
}

[Fact]
public void Index_Set_GetInvalidIndex_ThrowsArgumentOutOfRangeException()
{
    var attribute = new ExcelColumnIndexAttribute(-1);
    Assert.Throws<ArgumentOutOfRangeException>("value", () => attribute.Index = -1);
}
```

and to `ExcelColumnNameAttributeTests.cs`:
```
[Theory]
[InlineData("columnname")]
[InlineData("ColumnName")]
public void Name_Set_GetReturnsExpected(string value)
{
    var attribute = new ExcelColumnNameAttribute("Name")
    {
        Name = value
    };
    Assert.Equal(value, attribute.Name);
}

[Fact]
public void Name_SetNull_ThrowsArgumentNullException()
{
    var attribue = new ExcelColumnNameAttribute("Name");
    Assert.Throws<ArgumentNullException>("value", () => value.Name = null);
}

[Fact]
public void Name_SetEmpty_ThrowsArgumentNullException()
{
    var attribue = new ExcelColumnNameAttribute("Name");
    Assert.Throws<ArgumentException>("value", () => value.Name = string.Empty);
}

```

File: src/ExcelColumnNameAttribute.cs [hunk 1]
```diff
@@ -34,6 +34,6 @@ public ExcelColumnNameAttribute(string name)
         /// <summary>
         /// The name of the column.
         /// </summary>
-        public string Name { get; }
+        public string Name { get; set; }
```
[the remark this answers is not in the document]
Comment (yoldascevik) [reply src/ExcelColumnNameAttribute.cs]: I encapsulated Name property

File: tests/ExcelMapper/ExcelColumnIndexAttributeTests.cs [hunk 2]
```diff
@@ -17,7 +17,26 @@ public void Ctor_Int(int index)
         [Fact]
         public void Ctor_InvalidIndex_ThrowsArgumentOutOfRangeException()
         {
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => new ExcelColumnIndexAttribute(-1));
+            Assert.Throws<ArgumentOutOfRangeException>("value", () => new ExcelColumnIndexAttribute(-1));
```
Comment (hughbe) [inline tests/ExcelMapper/ExcelColumnIndexAttributeTests.cs:20]: this should still be "index" - could you add back the validation in the constructor

File: tests/ExcelMapper/ExcelColumnNameAttributeTests.cs [hunk 3]
```diff
@@ -17,13 +17,39 @@ public void Ctor_String(string name)
         [Fact]
         public void Ctor_NullName_ThrowsArgumentNullException()
         {
-            Assert.Throws<ArgumentNullException>("name", () => new ExcelColumnNameAttribute(null));
+            Assert.Throws<ArgumentNullException>("value", () => new ExcelColumnNameAttribute(null));
```
Comment (hughbe) [inline tests/ExcelMapper/ExcelColumnNameAttributeTests.cs:20]: Ditto, cheers

**Pull request merged**